validator/project_validator.go

```diff
@@ -518,13 +518,14 @@ func ensureReferentialIntegrity(project *model.Project, distroIds []string) Vali
 
 // validateTaskNames ensures the task names do not contain unauthorized characters.
 func validateTaskNames(project *model.Project) ValidationErrors {
+	unauthorizedTaskCharacters := unauthorizedCharacters + " "
```

Comment:
Does this cause any existing projects to break?

Reply:
Nope I ran it through all the configs and it looks good.